index.js

```diff
@@ -1,169 +1,189 @@
-            4) *
-          1.666;
-        // Seems to max at 6, so we're scaling it up to 10.
-        break;
-      case 'speed':
-        vibration = data.speed / 100;
-        break;
-      case 'power':
-        vibration = Math.abs(data.power) / 1000000;
-        break;
-      case 'torque':
-        vibration = Math.abs(data.torque) / 1000;
-        break;
-      case 'brake':
-        vibration = data.brake / 255;
-        break;
-      case 'accel':
-        vibration = data.accel / 255;
-        break;
-    }
-
-    let vibrationInt = Math.round(vibration * maxVibration);
-    if (vibrationInt > maxVibration) vibrationInt = maxVibration;
-
-  }
-  sendVibration(vibrationInt);
-};
-
-(async function () {
-  console.log("Options:");
-  console.log(chalk.blue.bold("rpm            ") + "- RPM from the Engine " + chalk.bold("(Best all around experience)"));
-  console.log(chalk.blue.bold("rumble-average ") + "- Surface Rumble on Wheels " + chalk.italic("(Averaged out)"));
-  console.log(chalk.blue.bold("speed          ") + "- Speed, slow but fun");
-  console.log(chalk.blue.bold("power          ") + "- Not quite sure, seems to be a similar thing to rpm.");
-  console.log(chalk.blue.bold("torque         ") + "- Torque of the Vehicle");
-  console.log(chalk.blue.bold("brake          ") + "- How hard are you pushing down on the Brake Pedal");
-  console.log(chalk.blue.bold("accel          ") + "- How hard are you pushing down on the Accel Pedal");
-
-  var vibrationFrom = null;
-  while (vibrationFrom === null) {
-    console.log(chalk.bold("Select an option for Toys to React to?"))
-    vibrationFrom = prompt( chalk.green("> ") );
-
-    if (!vibrationFrom || acceptedValues.indexOf(vibrationFrom) === -1) {
-      vibrationFrom = null;
-      console.warn(
-        chalk.yellow("Invalid Value selected, please select a value from the list above")
-      );
-    }
-  }
-
-  var maxVibration = null; // Max Power of 1-20 (Use 20 if ya wanna go wild)
-  while (maxVibration === null) {
-    console.log(chalk.bold("What is the Max Vibration you would like to have? (1 to 20)"))
-    var stringMax = prompt( chalk.green("> ") );
-    maxVibration = Number(stringMax);
-
-    if (isNaN(maxVibration) || maxVibration < 1) {
-      maxVibration = null;
-      console.warn(
-        chalk.yellow("Please provide a Number between 1 & 20")
-      );
-    }
-    else if (maxVibration > 20) {
-      maxVibration = 20;
-      console.warn(
-        chalk.grey("Number exceeds 20, Setting value to 20")
-      );
-    }
-  }
-
-  await forza.loadGames();
-  forza.startAllGameSockets();
-  forza.on('telemetry', throttle(processData, 100));
-})();
+import chalk from 'chalk';
+import ForzaJS from 'forza.js';
+import nodeFetch from 'node-fetch';
+import promptSync from 'prompt-sync';
+
+const forza = new ForzaJS.default();
+const prompt = promptSync({ sigint: true });
+
+// TODO: swap to using Got instead of NodeFetch. it's much better
```

Comment:
Also, because it has proper typescript definitions the ts server can actually understand what is going on even with it being JavaScript (which, really, should just swap to typescript